StreamChatClient_v3/APIClient/Endpoints/ChannelEndpoints_Tests.swift

```diff
@@ -36,11 +36,11 @@ final class ChannelEndpoints_Tests: XCTestCase {
         let channelID = ChannelId(type: .livestream, id: "qwerty")
```

Comment:
I think will a good practice to use everywhere `cid`, not `channelID` or `channelId`. We usually don't use `ID`, instead `Id` as a prefix to something else.

Reply:
Yep, @VojtaStavik told me about that.
Already using `cid` and made some renaming in this PR regarding this.